src/repo.json

```diff
@@ -4,7 +4,7 @@
     "repo_uri": "https://canonical.o3de.org",
     "summary": "The Open 3D Engine's canonical repos.",
     "additional_info": "Get more information about this repo at https://github.com/o3de/canonical.o3de.org",
-    "last_updated": "2023-10-09",
```

Comment:
The date needs to match when it went into dev?

Reply:
Hmmm we could update it today but even if I do that, the file might not get pushed to the bucket till another day - maybe something we should consider in the future, for now it's just important that the date advances so folks know there is a update.